src/server.js

```diff
@@ -13,7 +13,7 @@ LogUtils.log({ message: 'server.js: Initializing data and waiting for Graphhoppe
 
 const PORT: number = parseInt(process.env.PORT) || 80;
 const SERVER_ADDRESS: string = '0.0.0.0';
-const TWENTY_SECONDS_IN_MS: number = 20000;
+const FIVE_SECONDS_IN_MS: number = 5000;
```

Comment:
why do you have to save this in a constant? if you were going to abstract out a timeout time why wouldn't you name it something other than the word description of the numerical value?

Reply:
it's to explain what the value corresponds to, because if someone saw `5000` they might not be sure what 5000 represents -> this makes it explicit that it's referring to a specific number of milliseconds!

it's the difference between `f(5000)` and `f(FIVE_SECONDS_IN_MS)`, the second one is much more readable